js/search.js

```diff
@@ -22,9 +22,9 @@ $(document).ready(function() {
       });
     }
     $(".results__list").prepend(`
-      <p class="results__summary">
+      <h1><p class="results__summary">
```

Comment:
We can get rid of the `<p>` tag entirely on this, and just have it as `<h1 class="results__summary">...</h1>`

Reply:
The margins on `results__summary` will need some adjusting to override default h1 margins.